### DISCUSSION TITLE
如何支持flash游戏

### DISCUSSION THREAD

**Discussion by linkoln** [CefView/QCefView#360] [Q&A]
![image](https://github.com/CefView/QCefView/assets/5786476/4dbe134a-703c-425d-82df-4d39696eca03)

![image](https://github.com/CefView/QCefView/assets/5786476/f1390ae1-1705-434a-8310-251470bfd6d9)

加了这些选项都还是不行，求救

**Comment by L-Super** [comment]
Please confirm whether the current version of CEF supports flash.

**Comment by linkoln** [comment]
您好，您发来的邮件我已经收到，我会尽快阅读

**Comment by linkoln** [comment]
您好，您发来的邮件我已经收到，我会尽快阅读
